matlab/fiff_readout_eventlist.m

```diff
@@ -0,0 +1,98 @@
+function eventlist = fiff_readout_eventlist(fname, stim_chennal, consecutive, output)
+%	Find event from raw file
+%	
+%	Parameters
+%	----------
+%	fname : String, .fiff raw data file name
+%	stim_chennal : the channel that record event
+%   consecutive : bool | 'increasing'
```

Comment:
please fix indentation

Reply:
but use the way the doc formating is done in matlab see eg

https://github.com/mne-tools/mne-matlab/blob/master/matlab/mne_write_cov.m